plugins/kernel_v2/config/disk.rb

```diff
@@ -0,0 +1,151 @@
+require "log4r"
+require "securerandom"
+
+module VagrantPlugins
+  module Kernel_V2
+    class VagrantConfigDisk < Vagrant.plugin("2", :config)
+      #-------------------------------------------------------------------
+      # Config class for a given Disk
+      #-------------------------------------------------------------------
+
+      DEFAULT_DISK_TYPES = [:disk, :dvd, :floppy].freeze
```

Comment:
Maybe we don't need to be so rigid right now with these default disk types :man_shrugging:

Reply:
At this point I think this is acceptable for the first pass.